ice/server.py

```diff
@@ -0,0 +1,8 @@
+from uvicorn import main
+
+params_by_name = {p.name: p for p in main.params}
```

Comment:
How does this variable get used?

Reply:
It's not meant to be used externally.

Reply:
Never mind, just noticed the existing defaults are getting mutated